content/docs/glossary/multisig.mdx

```diff
@@ -69,10 +73,18 @@ This type of signer is especially useful in escrow accounts. You can pre-authori
 
 ### Hash(x)
 
-Adding a signature of type hash(x) allows anyone who knows `x` to sign the transaction. This type of signer is especially useful in [atomic cross-chain swaps](https://en.bitcoin.it/wiki/Atomic_cross-chain_trading) which are needed for inter-blockchain protocols like [lightning networks](https://lightning.network).
+Adding a signer of type hash(x) allows anyone who knows `x` to sign the transaction. This type of signer is especially useful in [atomic cross-chain swaps](https://en.bitcoin.it/wiki/Atomic_cross-chain_trading) which are needed for inter-blockchain protocols like [lightning networks](https://lightning.network).
 
 First, create a random 256 bit value, which we call `x`. The SHA256 hash of that value can be added as a signer of type hash(x). Then in order to authorize a transaction, `x` is added as one of the signatures of the transaction. Keep in mind that `x` will be known to the world as soon as a transaction is submitted to the network with `x` as a signature. This means anyone will be able to sign for that account with the hash(x) signer at that point. Often you want there to be additional signers so someone must have a particular secret key and know `x` in order to reach the weight threshold required to authorize transactions on the account.
 
+### Signed Payloads
+
+Adding a signed payload as a signer means that transactions with a signature on that payload by that signer are valid.
+
+More verbosely, you add a signer of `(public key, payload)`, where the public key is the Stellar address of the expected signer and the payload is any arbitrary data of up to 64 bytes. A valid signature, then, is a signature on that payload by the private key corresponding to the public key. This is also described in [CAP-40](https://stellar.org/protocol/cap-40) which introduced this new signer.
```

Comment:
```suggestion
More verbosely, you add a signer of `(public key, payload)`, where the public key is a Stellar public key of the expected signer and the payload is any arbitrary data of 1 to 64 bytes in length. A valid signature, then, is a signature of that payload by the private key corresponding to the public key. This is also described in [CAP-40](https://stellar.org/protocol/cap-40) which introduced this new signer.
```

Reply:
```suggestion
More verbosely, you add a signer of `(public key, payload)`, where the public key is the Stellar public key of the expected signer and the payload is 1 to 64 bytes of arbitrary data. A valid signature, then, is a signature of that payload by the private key corresponding to the public key. This is also described in [CAP-40](https://stellar.org/protocol/cap-40) which introduced this new signer.
```

Saying `address` is intended to explain that it's a `G...` value, but saying "where the public key is the Stellar public key" seems redundant...